src/components/MainDashboard/OpenPerscription/OpenPerscription.js

```diff
@@ -0,0 +1,86 @@
+import style from "./OpenPerscription.module.css";
+import Accordion from 'react-bootstrap/Accordion';
+
+const OpenPerscription = () => {
+    return (
+        <section className={style["section-openPerscription"]}>
+            <div className={style["div-main-content"]}>
+                <div className={style["div-header"]}>
+                    <h2 className={style["div-header-mainTitile"]}>Рецептурна Бланка</h2>
+                    <h3 className={style["div-header-title"]}>Министерство на здравеопазването</h3>
+                    <p>Д-р: Радева</p>
+                    <p>УИН: 0000000000</p>
+                    <p>&#8470; Лек. Практика: 0000000000</p>
+                    <p>Телефон: 089xxxxxxx</p>
+                </div>
```

Comment:
Please use the provided style properties from the design for all of the above texts, as well as for the patient's info